VideoAPI/VideoApi.DAL/Commands/AddMessageCommand.cs

```diff
@@ -42,6 +43,11 @@ public async Task Handle(AddMessageCommand command)
                 throw new ConferenceNotFoundException(command.ConferenceId);
             }
 
+            if(!conference.DoesParticipantExist(command.From) && !conference.DoesParticipantExist(command.To))
```

Comment:
have you added to the AddMessageCommand Tests to cover these scenarios?

Reply:
Added